packages/proot-distro/build.sh

```diff
@@ -23,5 +23,5 @@ etc/proot-distro/ubuntu-20.04.sh
 "
 
 termux_step_make_install() {
-	./install.sh
+	TERMUX_PREFIX="$TERMUX_PREFIX" TERMUX_ANDROID_HOME="$TERMUX_ANDROID_HOME" ./install.sh
```

Comment:
I'm the author of `proot-distro` package and I'm sure that this is completely unnecessary here. Variables are already exported by build-package.sh.

Reply:
Hm no these variables are set but not exported
https://github.com/termux/termux-packages/blob/master/scripts/properties.sh#L19